terraform_files/nutanix-ci-machine/variables-nutanix.tf

```
@@ -0,0 +1,67 @@
+//////
+// Nutanix variables
+//////
+
+variable "nutanix_endpoint" {
+  type        = string
+  description = "Endpoint for the Prism Elements or Prism Central instance. This can also be specified with the NUTANIX_ENDPOINT environment variable"
+}
+
+variable "nutanix_username" {
+  type        = string
+  description = "Username for the Prism Elements or Prism Central instance. This can also be specified with the NUTANIX_USERNAME environment variable"
+}
+
+variable "nutanix_password" {
+  type        = string
+  description = "Password for the Prism Elements or Prism Central instance. This can also be specified with the NUTANIX_PASSWORD environment variable"
+}
+
+variable "nutanix_port" {
+  type        = number
+  description = "Port for the Prism Elements or Prism Central instance. This can also be specified with the NUTANIX_PORT environment variable. Defaults to 9440"
+}
+
+variable "nutanix_cluster" {
+  type        = string
+  description = "Nutanix cluster name"
+}
+
+
+variable "nutanix_subnet" {
+  type        = string
+  description = "Nutanix subnet name. While selected the nic Ip will be in that subnet addresses range"
+}
+
+variable "iso_name" {
+  type        = string
+  description = "Assisted Test Infra machine image name"
+}
+
+variable "build_id" {
+  type        = string
+  description = "The CI build id"
+}
+
+variable "memory" {
+  type = number
+  default = 8192
+  description = "RAM in MiB allocated to masters"
+}
+
+variable "vcpu" {
+  type = number
+  default = 2
```

It doesn't seem like enough, maybe change it to 4/8

```suggestion
  default = 4
```